src/lib/math.js

```diff
@@ -314,6 +314,55 @@ const $builtinmodule = function (name) {
         throw new Sk.builtin.NotImplementedError("math.isqrt() is not yet implemented in Skulpt");
     };
 
+    function lcm() {
+        const nargs = arguments.length;
+
+        // lcm() without arguments returns 1
+        if (nargs === 0) return new Sk.builtin.int_(1);
+
+        let result = arguments[0];
+        Sk.builtin.pyCheckType("*integers", "integer", Sk.builtin.checkInt(result));
```

Comment:
Adding same thing in a loop (`const argument = Sk.misceval.asIndexOrThrow(args[i]);`). It breaks tests with `Error: Assertion failure: JS object passed to richCompareBool`. Should I even use it there? It has same checks as for first argument.

Reply:
Yes you should. This means that a javascript object is being leaked into python.

It's because you're returning `0` on line 348 (rather than a python integer). 

In the original version that code path never hit because at that stage `argument` was a python object so the test
`if (argument === 0)` was always falsey.